mite/context.py

```diff
@@ -53,25 +54,37 @@ def should_stop(self):
             return self._should_stop_func()
         return False
 
+    @property
+    def transaction_name(self):
+        return active_transaction.get()[0]
+
+    @property
+    def transaction_id(self):
```

Comment:
I think this wants to have an underscore-prefixed name since it's an implementation detail that users of mite shouldn't care about